.github/workflows/CI.yml

```
@@ -0,0 +1,54 @@
+      - name: Build solution
+        run: dotnet build --no-incremental backend/Employee.API
+        
+      - name: Set environment variable for directory
+        run: echo "PROJECT_DIR=backend/Employee.API/EmployeeXUnit.Test" >> $GITHUB_ENV
+
+      # 6. Collect test coverage with dotnet-coverage
+      - name: Collect coverage using dotnet-coverage
+        run: |
+          dotnet-coverage collect "dotnet test" -f xml -o "coverage.xml"
+        working-directory: ${{ env.PROJECT_DIR }}
+        
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)